### PR TITLE
Added support for sync CLI

### PR DESCRIPTION
**Pull request by prklm10**
- adding timeout to 600 sec.
- sync implementation

### CODE REVIEW

**Review by shantanuk-browserstack — APPROVED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,100 @@
+GEM
```
Comment (chinmay-browserstack) [inline Gemfile.lock:1]: Remove this file + changes in Gemfile

File: percy/lib/cli_wrapper.rb [hunk 2]
```diff
@@ -49,13 +49,14 @@ def self.percy_enabled?
     end
 
     def post_screenshots(name, tag, tiles, external_debug_url = nil, ignored_elements_data = nil,
-                        considered_elements_data = nil)
-      body = request_body(name, tag, tiles, external_debug_url, ignored_elements_data, considered_elements_data)
+                         considered_elements_data = nil, sync = false)
```
Comment (chinmay-browserstack) [inline percy/lib/cli_wrapper.rb:52]: Remove default value

File: percy/providers/app_automate.rb [hunk 3]
```diff
@@ -29,9 +30,10 @@ def screenshot(name, **kwargs)
       begin
         response = super(name, **kwargs)
         percy_screenshot_url = response.fetch('link', '')
-        execute_percy_screenshot_end(name, percy_screenshot_url, 'success')
+        execute_percy_screenshot_end(name, percy_screenshot_url, 'success', sync)
```
Comment (chinmay-browserstack) [inline percy/providers/app_automate.rb:33]: no need to pass sync here

File: Gemfile.lock (hunk 1 above)
Comment (prklm10) [reply Gemfile.lock]: removed

File: percy/lib/cli_wrapper.rb (hunk 2 above)
Comment (prklm10) [reply percy/lib/cli_wrapper.rb]: default to null.

File: percy/providers/app_automate.rb (hunk 3 above)
Comment (prklm10) [reply percy/providers/app_automate.rb:33]: it is for sync call instrumentation.

**Pull request closed without merging**